pyanaconda/modules/boss/boss_interface.py

```diff
@@ -84,3 +84,10 @@ def DistributeKickstart(self) -> List[Dict[Str, Variant]]:
             "line_number": get_variant(Int, result["line_number"]),
             "error_message": get_variant(Str, result["error_message"])
         } for result in results]
+
+    def GetModules(self) -> List[Str]:
```

Comment:
Please, add a unit test for this method.

Reply:
I removed the method